src/util/slateHelpers.js

```diff
@@ -68,6 +67,42 @@ const RULES = [
   {
     // Aside handling
```

Comment:
Blir vel feil å kalle dette for aside handling?

Reply:
Jo. Fikser.

Reply:
Fikset